orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisActiveExecutionsMonitor.kt

```diff
@@ -45,12 +48,14 @@ import java.util.concurrent.atomic.AtomicLong
  *
  * TODO rz - Add cloudProviders, accounts as tags
  */
-
+@Component
 class RedisActiveExecutionsMonitor(
   private val executionRepository: ExecutionRepository,
   @Qualifier("jedisPool") private val pool: Pool<Jedis>,
   private val objectMapper: ObjectMapper,
-  private val registry: Registry
+  private val registry: Registry,
+  @Value("\${queue.monitor.activeExecutions.refresh.frequency.ms:60000}") refreshFrequencyMs: Long,
```

Comment:
@robzienert Changed the config key name here ... I cannot imagine anyone is actually using it so figured it was ok (s/register/refresh)

Reply:
💯